src/ifcopenshell-python/ifcopenshell/api/alignment/__get_segment_start_point_label.py

```diff
@@ -0,0 +1,251 @@
+                    "HELMERTCURVE": "xx",
+                    "LINE": "S.T.",
+                    "SINECURVE": "xx",
+                    "VIENNESEBEND": "xx",
+                },
+                "CUBIC": {
+                    "BLOSSCURVE": "xx",
+                    "CIRCULARARC": "S.C.",
+                    "CLOTHOID": "xx",
+                    "COSINECURVE": "xx",
+                    "CUBIC": "xx",
+                    "HELMERTCURVE": "xx",
+                    "LINE": "S.T.",
+                    "SINECURVE": "xx",
+                    "VIENNESEBEND": "xx",
+                },
+                "HELMERTCURVE": {
+                    "BLOSSCURVE": "xx",
+                    "CIRCULARARC": "S.C.",
+                    "CLOTHOID": "xx",
+                    "COSINECURVE": "xx",
+                    "CUBIC": "xx",
+                    "HELMERTCURVE": "xx",
+                    "LINE": "S.T.",
+                    "SINECURVE": "xx",
+                    "VIENNESEBEND": "xx",
+                },
+                "LINE": {
+                    "BLOSSCURVE": "T.S.",
+                    "CIRCULARARC": "P.C.",
+                    "CLOTHOID": "T.S.",
+                    "COSINECURVE": "T.S.",
+                    "CUBIC": "T.S.",
+                    "HELMERTCURVE": "T.S.",
+                    "LINE": "P.I.",
+                    "SINECURVE": "T.S.",
+                    "VIENNESEBEND": "T.S.",
+                },
+                "SINECURVE": {
+                    "BLOSSCURVE": "xx",
+                    "CIRCULARARC": "S.C.",
+                    "CLOTHOID": "xx",
+                    "COSINECURVE": "xx",
+                    "CUBIC": "xx",
+                    "HELMERTCURVE": "xx",
+                    "LINE": "S.T.",
+                    "SINECURVE": "xx",
+                    "VIENNESEBEND": "xx",
+                },
+                "VIENNESEBEND": {
+                    "BLOSSCURVE": "xx",
+                    "CIRCULARARC": "S.C.",
+                    "CLOTHOID": "xx",
+                    "COSINECURVE": "xx",
+                    "CUBIC": "xx",
+                    "HELMERTCURVE": "xx",
+                    "LINE": "S.T.",
+                    "SINECURVE": "xx",
+                    "VIENNESEBEND": "xx",
+                },
+            }
+            label = lookup_table[prev_segment.DesignParameters.PredefinedType][segment.DesignParameters.PredefinedType]
+        elif segment.DesignParameters.is_a("IfcAlignmentVerticalSegment"):
+            lookup_table = {
+                "CIRCULARARC": {"CIRCULARARC": "xx", "CLOTHOID": "xx", "CONSTANTGRADIENT": "xx", "PARABOLICARC": "xx"},
+                "CLOTHOID": {"CIRCULARARC": "xx", "CLOTHOID": "xx", "CONSTANTGRADIENT": "xx", "PARABOLICARC": "xx"},
+                "CONSTANTGRADIENT": {
+                    "CIRCULARARC": "xx",
+                    "CLOTHOID": "xx",
+                    "CONSTANTGRADIENT": "P.V.I",
+                    "PARABOLICARC": "B.V.C.",
```

Comment:
```suggestion
                    "PARABOLICARC": "P.V.C.",
```
point of vertical curvature

Reply:
Revised as noted